### DISCUSSION TITLE
orderNumber가 필요해 보입니다.

### DISCUSSION THREAD

**Discussion by wenodev** [CodeSoom/DDD-Kurly-Clone-Order#14] [Ideas]
- Order 혹은 OrderSheet에 orderNumber가 필요하다고 생각합니다.
![image](https://user-images.githubusercontent.com/35131529/139003964-f52f8ab9-aa15-4733-af5d-6a5f196d2ced.png)

**Comment by joowankim** [comment]
orderNumber는 무엇을 위함인가요? 혹시, 식별자를 말씀하시는 거라면 Order에 이미 orderId를 부여하고 있습니다. 아니면 다른 이유가 있으신가요?
